extensions/ql-vscode/src/remote-queries/view/RemoteQueries.tsx

```diff
@@ -59,6 +68,14 @@ export function RemoteQueries(): JSX.Element {
   }
 
   try {
+    const openQueryFile = () => {
+      openFile(queryResult.queryPath);
```

Comment:
What happens if the file is deleted before the results come back? Does VS Code handle it gracefully?